src/loader.c

```diff
@@ -338,6 +338,8 @@ bool parse_database(void *base, const char *name, const char *connstr)
 			min_pool_size = atoi(val);
 		} else if (strcmp("reserve_pool", key) == 0) {
```

Comment:
Would be good to add a comment here that we continue supporting this for backwards compatability.